pipeline.yml

```diff
@@ -7,14 +7,14 @@ jobs:
   - task: run-compliance-masonry
     config:
       platform: linux
-      image: docker:///geramirez/compliance-masonry-cli-build#0.4
+      image: docker:///geramirez/compliance-masonry-go#0.2
```

Comment:
Where is the Dockerfile for this? Can we switch it to be an [automated build](https://docs.docker.com/docker-hub/github/) under the [18fgsa organization on Docker Hub](https://hub.docker.com/u/18fgsa/dashboard/)?